.deepsource.toml

```diff
@@ -0,0 +1,25 @@
+version = 1
+
+test_patterns = ["tests/**"]
+
+[[analyzers]]
+name = "python"
+enabled = true
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
+
+[[analyzers]]
+name = "docker"
+enabled = true
+
+  [analyzers.meta]
+  dockerfile_paths = ["Dockerfile"]
+
+[[analyzers]]
+name = "javascript"
+enabled = true
+
+  [analyzers.meta]
+  environment = ["browser"]
+
```

Comment:
Let's remove this please -- we don't want to add yet another scanner

Reply:
@kaxil  have  removed `.deepsource.toml` file.
Thanks